test/images/node-perf/npb-ep/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
+
+RUN apt-get update && apt-get install -y build-essential gfortran
```

Comment:
can you use a multi-stage build so that we don't get all of the build tools in the final image?

Reply:
see `test/images/sample-apiserver/Dockerfile` for an example of using multi-stage